pkg/build/builder/docker.go

```diff
@@ -210,6 +214,11 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	instructions := dockerfile.ParseTreeToDockerfile(node)
 
 	// Overwrite the Dockerfile.
+	f, err := os.Open(dockerfilePath)
```

Comment:
You can just use a numeric octal value that corresponds to a linux mode (ie `0644`).